cli/util/util.go

```diff
@@ -920,7 +920,7 @@ type AppListEntry struct {
 }
 
 // CollectAppList collects all the supposed applications in passed appsPath directory.
-func CollectAppList(baseDir string, appsPath string) ([]AppListEntry, error) {
+func CollectAppList(baseDir string, appsPath string, verbose bool) ([]AppListEntry, error) {
```

Comment:
Why do you need this change?
Move changes to the separate commit (the change in many places makes it difficult to see the meaningful code).

Reply:
Moved. I need it, since CollectAppList prints to the output messages that are not needed for tt list, but is needed for other commands.